homepage/homepage/content/docs/server-side/setup.mdx

```diff
@@ -117,3 +117,36 @@ Currently, the Jazz Loader is tested on the following edge environments:
 
 - Edge runtime environment that supports WebAssembly
 - `jazz-tools/load-edge-wasm` must be imported before any Jazz import
+
+
+## Node-API
+
+Jazz uses a WASM-based crypto implementation for maximum compatibility and performance across a wide variety of environments. 
```

Comment:
Well, N-API is faster than WASM so WASM doesn't offer maximum performance 😆 

I'd say that it provides almost native performance.
N-API offers native performance.

Reply:
Sorry, that's my regression there :) I wanted to highlight performance across a wide variety of environments vs performance on a more limited set of environments. Will make a new suggestion!